code/modules/mob/living/simple_animal/abnormality/waw/flesh_idol.dm

```diff
@@ -0,0 +1,73 @@
+/mob/living/simple_animal/hostile/abnormality/flesh_idol
+	name = "Flesh Idol"
+	desc = "A cross with flesh stapled in the middle."
+	icon = 'ModularTegustation/Teguicons/64x96.dmi'
+	icon_state = "flesh_idol"
+	maxHealth = 600
+	health = 600
+	threat_level = WAW_LEVEL
+	pixel_x = -16
+	base_pixel_x = -16
+	work_chances = list(
+		ABNORMALITY_WORK_INSTINCT = 40,
+		ABNORMALITY_WORK_INSIGHT = 40,
+		ABNORMALITY_WORK_ATTACHMENT = 40,
+		ABNORMALITY_WORK_REPRESSION = 40
+			)
+	start_qliphoth = 1
+	work_damage_amount = 10
+	work_damage_type = RED_DAMAGE
```

Comment:
Same here. Change the damage proc to be random type.

Reply:
I converted it to red across the board to keep it consistent. Making it random would make it more difficult on top of the already brutal effect it has

Reply:
That was basically the point, though. We do not need yet another red damage abnormality, we have a ton of these.

Reply:
I think the point was the healing on top of breaching if you went too far.... the damage type is kinda just a footnote. 
In terms of Red damage, we actually lack a lot of late game red damage, especially during work.
There's:
Nothing There
Mountain of Smiling Bodies (ONLY when it breaches and only it's normal attack)
Queen bee
General Bee
Dreaming Current (ONLY when it breaches)
King of Greed, I guess? 

That's 4 and two halves of abnormalities out of 18. The Majority of WAW+ does black.

Reply:
Egor, I've spoken with the other devs, I'm NOT making this random

Reply:
There's only one maintainer other than you, and that's Redacted. And as far as I can tell, they too thought of making it random as a good idea.
And development isn't democracy either.

Reply:
It was mostly me speaking to the juniors. I dunno, I just think that would make this abnormality unworkable. Entirely removing the preparation is a bad idea.